pkg/bigint/bigint_test.go

```diff
@@ -0,0 +1,24 @@
+package bigint_test
+
+import (
+	"encoding/json"
+	"github.com/ethersphere/bee/pkg/bigint"
+	"reflect"
+	"testing"
+)
+
+type TestStruct struct {
+	Bg *bigint.BigInt `json:"bg"`
+}
+
+func TestMarshaling(t *testing.T) {
+	mar, err := json.Marshal(TestStruct{Bg: bigint.NewBigInt(5)})
```

Comment:
i would add tests for values which are higher than `math.MaxInt64`, since we are testing `big.Int`s here